spec/system/candidate_interface/entering_details/candidate_entering_other_qualification_btec_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering their other qualifications' do
+  include CandidateHelper
+
+  scenario 'Candidate submits their BTEC qualification' do
+    given_i_am_signed_in
+    and_i_visit_the_site
+
+    when_i_click_on_other_qualifications
+    then_i_see_the_select_qualification_type_page
+
+    when_i_select_add_an_other_uk_qualification
+    and_i_fill_in_the_qualification_name_as_btec
+    and_i_click_continue
+    then_i_see_the_add_btec_qualifications_form
+
+    when_i_fill_in_some_of_my_qualification_but_omit_some_required_details
+    and_i_submit_the_other_qualification_form
+    then_i_see_validation_errors_for_my_qualification
+
+    when_i_select_a_grade
+    and_i_submit_the_other_qualification_form
+    then_i_see_the_other_qualification_review_page
+    and_i_should_see_my_qualifications
```

Comment:
Perhaps consolidate the above 4 into:
`when_i_complete_the_form`
`then_i_see_my_btec_on_the_review_page`

Reply:
Thanks, I've made those changes.

For future reference, is there a guiding principle you use here for when to group actions together or to spell them out explicitly? 

I was following the existing patterns from `candidate_entering_other_qualification_spec.rb`.

Reply:
> is there a guiding principle you use here for when to group actions together or to spell them out explicitly?Nothing that's been agreed across the team, but I tend to err towards the minimum number of steps

Reply:
I think this is something we've generally been very lax and haven't had rules about. I'm personally a fan of grouping, although I'm def guilty of not having done it in the past.

Reply:
>is there a guiding principle you use here for when to group actions together or to spell them out explicitly?

Nothing that's been agreed across the team, but I tend to err towards the minimum number of steps I feel I can get away with while still being clear about what behaviour is being tested. Keeps the top-level description brief and readable, while the detail is available if needed further down.

Reply:
Nice, thanks 👍